media-gfx/solvespace/solvespace-2.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DXFRW_COMMIT="87ff1082f49b1286a033ac8f38fe8dde0975bb8b"
+DXFRW_PV="0.6.3"
+DXFRW_PN="libdxfrw"
+DXFRW_P="${DXFRW_PN}-${DXFRW_PV}-${DXFRW_COMMIT}"
+
+inherit cmake-utils
+
+DESCRIPTION="Parametric 2d/3d CAD"
+HOMEPAGE="http://solvespace.com"
+SRC_URI="https://github.com/solvespace/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/solvespace/${DXFRW_PN}/archive/${DXFRW_COMMIT}.zip -> ${DXFRW_P}.zip"
+
+LICENSE="GPL-3"
```

Comment:
Could you verify whether copyright notices on top of source files don't specify '[...] Public License version 3 or (at your option) newer' or something similar? If so, please make it `GPL-3+`.

Reply:
Nah. Unless you can find somewhere where program authors explicitly say 'or newer version', let's leave it as GPL-3.